Source/VCExpr/VCExpressionGenerator.cs

```diff
@@ -445,6 +445,7 @@ int AndSize(VCExpr e)
     public static readonly VCExprOp MinimizeOp = new VCExprCustomOp("minimize##dummy", 2, Type.Bool);
     public static readonly VCExprOp MaximizeOp = new VCExprCustomOp("maximize##dummy", 2, Type.Bool);
     public static readonly VCExprOp NamedAssumeOp = new VCExprCustomOp("named_assume##dummy", 2, Type.Bool);
+    public static readonly VCExprOp NamedAssertOp = new VCExprCustomOp("named_assert##dummy", 2, Type.Bool);
```

Comment:
It's a little awkward: they're inserted into the formula as placeholders and then removed while converting to SMT-Lib. I can see if it's possible to do any refactoring to avoid that.

Reply:
See my comment in the previous thread.

Reply:
If you don't end up changing the code, could you add comments to the code that explain how they work?